apps/atr-one/app/frontend/components/Visualizer.vue

```diff
@@ -2,8 +2,10 @@
 
 import LoggingPanel from './LoggingPanel.vue'
 import TimelinePanel from './TimelinePanel.vue'
-import { subscribeToPubnub } from '../helpers/pubnub-wrapper'
+// import { subscribeToPubnub } from '../helpers/pubnub-wrapper'
```

Comment:
Use centrifugo instead of pubnub. Again, these helpers could perhaps be combined into one.